### PR TITLE
Remove user from organization implemented

### PR DESCRIPTION
**Pull request by mdrusko1**

### CODE REVIEW

File: src/app/organizations/organization-details/organization-details.component.ts [hunk 1]
```diff
@@ -126,4 +127,25 @@ export class OrganizationDetailsComponent implements OnInit {
...
+        SpinnerUtil.showSpinner();
+        this.organizationService.removeMemberFromOrganization(orgID, memberID)
+            .subscribe(() => {
+                    SpinnerUtil.hideSpinner();
+                    swal({
+                        title: '',
+                        text: 'Success!',
+                        type: 'success'
+                    }).then(function () {
+                        this.reloadPage('/dash/manage_groups/' + this.organization.uuid);
```
Comment (mightymatth) [inline src/app/organizations/organization-details/organization-details.component.ts:141]: Use string interpolation (<code>`/dash/manage_groups/${this.organization.uuid}`</code>).
As we talked previously, you don't need to reload the entire page. Rather refresh users with `getMembersForOrganization()` method.

File: src/app/shared/services/project/organization.service.ts [hunk 2]
```diff
@@ -50,6 +50,10 @@ export class OrganizationService {
     getMembersForOrganization(orgID: string) {
         return this.http.get<OrganizationMembersResponse>(`/api/project/organization/${orgID}/members`);
     }
+
+    removeMemberFromOrganization(orgID: string, memberID: string) {
+        return this.http.delete<void>(`/api/project/organization/${orgID}/members/${memberID}`, {});
```
Comment (mightymatth) [inline src/app/shared/services/project/organization.service.ts:55]:
```suggestion
        return this.http.delete<void>(`/api/project/organization/${orgID}/members/${memberID}`);
```
The second parameter is optional and therefore not needed.

File: src/app/organizations/organization-details/organization-details.component.ts [hunk 3]
```diff
@@ -126,4 +127,25 @@ export class OrganizationDetailsComponent implements OnInit {
...
+                    }.bind(this));
+                },
+                hideSpinnerAndDisplayError);
+    }
+
+    reloadPage(uri: string) {
+        this.router.navigateByUrl('/', {skipLocationChange: true}).then(() =>
+            this.router.navigate([uri]));
```
Comment (mightymatth) [inline src/app/organizations/organization-details/organization-details.component.ts:149]: Even though, this reloadPage is unnecessary in this case, why you do this promise chain with `navigateByUrl` and `navigate` chain?

**Review by mightymatth — CHANGES_REQUESTED** [review]

File: src/app/organizations/organization-details/organization-details.component.ts (hunk 1 above)
Comment (mdrusko1) [reply src/app/organizations/organization-details/organization-details.component.ts:141]: OK

**Review by mightymatth — APPROVED** [review]

**Pull request merged**